generators/server/templates/src/app.js

```diff
@@ -2,16 +2,20 @@ import express from "express";
 import path from "path";
 import logger from "morgan";
 import bodyParser from "body-parser";
-import "./src/boot.js";
-import configuration from "./src/config.js";
-import addMiddlewares from "./src/middlewares/addMiddlewares.js";
+import "./boot.js";
+import configuration from "./config.js";
+import addMiddlewares from "./middlewares/addMiddlewares.js";
+import { fileURLToPath } from "url"
```

Comment:
conventionally and with some VSCode tooling, we prefer to have the library related imports above the relative path imports for clarity